MediaBrowser.Model/Entities/MetadataProvider.cs

```diff
@@ -7,6 +7,12 @@ namespace MediaBrowser.Model.Entities
     /// </summary>
     public enum MetadataProvider
     {
+        /// <summary>
+        /// This provider is an escape hatch for the user to override the series
+        /// merging.
+        /// </summary>
+        User = 0,
```

Comment:
Wouldn't it be better to call it `Custom`?

Reply:
That wpuød probably be better. Will redo it later today

Reply:
And the xmldoc is also a bit too Series specific. It can technically be used by all entities.

Reply:
Updated the enum name and added a more generic description.